schedule/yam/agama_auto_lvm_s390x.yaml

```diff
@@ -0,0 +1,16 @@
+---
+name: agama unattended lvm
```

Comment:
please file a ticket to think about how to unify in single yaml `agama_auto_lvm.yaml` and `agama_auto_lvm_s390x.yaml` (with new names of course) so we can avoid this repetition in the future, remark what is different.

Reply:
ok, I'll file ticket for it.